docs/src/main/sphinx/connector/memsql.rst

```diff
@@ -28,7 +28,7 @@ connection properties as appropriate for your setup:
 .. code-block:: text
 
     connector.name=memsql
-    connection-url=jdbc:mariadb://example.net:3306
+    connection-url=jdbc:singlestore://example.net:3306
```

Comment:
Do any of the above docs need to be changed? Since we do not use the mariadb/mysql driver anymore maybe this bit should be changed?
```
The SingleStore connector
is very similar to the MySQL connector with the only difference being the
underlying driver.
```

Reply:
That text should go away entirely IMO. It isn't actionable at all.

Reply:
Can ye open an issue to track that as a separate change? I'm in favour of it / I just don't want to attempt it as part of this change.

Reply:
It wasn't visible in the diff, but if I should simply remove the reference to MySQL, I can add it to this commit (I'm already doing doc changes, so it fits - just don't want to have too many parallel discussions on 1 pr).
```
The SingleStore (formerly known as MemSQL) connector allows querying and
creating tables in an external SingleStore database. ~~~The SingleStore connector
is very similar to the MySQL connector with the only difference being the
underlying driver.~~~
```
👍  👎 on removing the last sentence?